django/chat/templates/chat/components/chat_security_label.html

```diff
@@ -12,14 +10,12 @@
   <ul class="dropdown-menu">
     <li class="dropdown-header">{% trans "Set security label" %}</li>
     {% for security_label in security_labels %}
- 
       <li>
         <button class="dropdown-item"
                 hx-get="{% url 'chat:set_security_label' chat.id security_label.id %}"
-                hx-target="closest div.dropdown"
+                hx-target="closest .dropdown"
                 hx-swap="outerHTML"
-                onclick="this.closest('div.dropdown').querySelector('button').innerHTML = '...';"
-                {% if security_label == chat.security_label %}disabled{% endif %}>
+                {% if security_label.id == chat.security_label.id %}disabled{% endif %}>
```

Comment:
No difference between comparing the object and the ID for Django objects:
https://medium.com/django-unleashed/effectively-comparing-django-model-instances-7ffe3548f19a